src/tools/batchrun/main.cpp

```diff
@@ -285,32 +160,25 @@ int main(int argc, char* argv[])
                 else
                     cmd << "call "; // why is it required for working ???
                 cmd << "\"" << solver << "\"";
-                if (optForce)
-                    cmd << " --force";
-                if (optForceExpansion)
-                    cmd << " --economy";
-                if (optForceEconomy)
-                    cmd << " --economy";
-                if (optForceAdequacy)
-                    cmd << " --adequacy";
-                if (!(!optName))
-                    cmd << " --name=\"" << *optName << "\"";
-                if (!(!optYears))
-                    cmd << " --year=" << *optYears;
-                if (optNoOutput)
-                    cmd << " --no-output";
-                if (optYearByYear)
-                    cmd << " --year-by-year";
-                if (optNoTSImport)
-                    cmd << " --no-ts-import";
-                if (optIgnoreAllConstraints)
-                    cmd << " --no-constraints";
-                if (optParallel)
-                    cmd << " --parallel";
-                if (optForceParallel)
-                    cmd << " --force-parallel=" << *optForceParallel;
-                if (ortoolsUsed)
-                    cmd << " --use-ortools --ortools-solver=" << ortoolsSolver;
+
+                // argv[0] => executable name
+                // argv[1] => directory name
+                for (int ii = 2; ii < argc; ii++)
```

Comment:
**-i / --input** is no longer available as a **batchrun** command line option.
Is this intended ?

We now imply that the search directory is necessarily the first argument the user will give, and without the **-i** introduction.
I'm not sure of that.

I'm tempted to suggest the following command line pattern : 
`./batchrun -i <search dir> <the other args>`, if it's doable.